### ISSUE TITLE
Suggestions for QoL Improvements

### ISSUE THREAD

**Issue by TheAymen** [Jenrykster/animeo#20]
Thank you for this useful add-on.

I would like to point out some things I noticed as a user that would further enhance the add-on if taken into consideration.

- Firstly, what I noticed is that watching anime from the Animeo (Anilist) catalogues (such as Watching section) does not register the episode as tracked (when tracking is configured for Kitsu entries only).

Secondly, I am not sure if this is possible from a Stremio add-on development POV,  but if it is, it would greatly enhance the add-on QoL, in my opinion. 

- If configuration were to allow for anime to be tracked only after the episode is finished, I think that would be useful because sometimes people may not watch an episode till the end and therefore wouldn't want the episode to be tracked or the series to be moved to completed on Anilist before actually finishing the series/episode.

Thank you.

**Comment by Pigamer37** [comment]
Hey @TheAymen! About the second point, that would be great but Stremio doesn't provide a way to let an addon know when a user has finished seeing the stream. In fact, everything Stremio requests to addons is sent before the user watches an item, the subtitle requests being the latest resource to be requested.

TL,DR in layman terms: there's no official way to know when a user has finished an episode, and the closest but kind of hacky way is the one implemented now, using subtitle requests as a signal for a user opening a stream.

**Comment by TheAymen** [comment]
Noted @Pigamer37. Do you think anything can be done for the first suggestion? I think it would be more internally consistent with the addon to have the items in the Animeo catalogs (Watching, Plan to Watch, etc) be Kitsu based, so that they track.

Thank you.
